bin/README.md

```diff
@@ -0,0 +1,18 @@
+The `exercise-gen.pl6` file must currently be run from within an exercise directory. It will retrieve data from an `example.yaml` file within that directory, and use the contained information to generate a test file using `templates/test.mustache`. If it finds a `canonical-data.json` file in `x-common` for the exercise in question it will be included.
```

Comment:
I've updated the docs to be a little clearer. The generator does not yet do multiple exercises, it simply uses the `example.yaml` file in the directory you're currently in, and figures out what canonical data you need for the exercise based on the directory name.